packages/core/src/types.ts

```diff
@@ -100,6 +100,7 @@ export type NormalizedOverrideOutput = {
   useDates?: boolean;
   useTypeOverInterfaces?: boolean;
   useDeprecatedOperations?: boolean;
+  useNumberTypeForBigInt?: boolean;
```

Comment:
I would do useBigInt and only generate bigInt when it's set to true. What do you think? Cause indeed right now it's causing a bit a breaking change with this

Reply:
Yeah, I agree, that way it will support both needs and not break all the frontends that are currently using it, haha. I pushed this change!